### PR TITLE
[BPK-1915] Add plugin for publishing

### PR DESCRIPTION
**Pull request by amolgupta**
Fixing the publishing configuration.

1. Add missing publishing plugin to library
2. Convert dependency to all lowercase

### CODE REVIEW

File: Backpack/build.gradle [hunk 1]
```diff
@@ -1,9 +1,10 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'org.jetbrains.dokka-android'
+apply plugin: 'com.github.dcendents.android-maven'
```
Comment (matthewdavidson) [inline Backpack/build.gradle]: What does this do?

File: Backpack/build.gradle [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 
 version = rootProject.ext.version
-group='com.github.Skyscanner.backpack-android'
+group='com.github.skyscanner.backpack-android'
```
Comment (matthewdavidson) [inline Backpack/build.gradle]: Why did this change from:
```
net.skyscanner.backpack:Backpack
```
to
```
com.github.Skyscanner:backpack-android
```

File: Backpack/build.gradle (hunk 1 above)
Comment (amolgupta) [reply Backpack/build.gradle]: Adds publishing plugin. see https://jitpack.io/docs/ANDROID/

File: Backpack/build.gradle (hunk 2 above)
Comment (amolgupta) [reply Backpack/build.gradle]: Moved from `Bintray` to `jitpack`. see https://gojira.skyscanner.net/browse/BPK-1858

Comment (matthewdavidson) [reply Backpack/build.gradle]: Bu why does the names space need to change?

Comment (amolgupta) [reply Backpack/build.gradle]: this is how the repository is located by jitpack.
 ` implementation 'com.github.User:Repo:Version'  `

Comment (matthewdavidson) [reply Backpack/build.gradle]: Gotcha 👍

**Pull request merged**